client/blocks/signup-form/index.jsx

```diff
@@ -611,7 +611,7 @@ class SignupForm extends Component {
 			return (
 				<p className="signup-form__terms-of-service-link">
 					{ this.props.translate(
-						'By creating an account via any of the options below, you agree to our {{a}}Terms of Service{{/a}}.',
+						'By creating an account, you agree to our {{a}}Terms of Service{{/a}}.',
```

Comment:
Hi! I've found a possible matching string that has already been translated **26** times:
`translate( 'By creating an account you agree to our {{a}}Terms of Service{{/a}}.' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
I guess we should remove the `,`...